backend/services/implementations/__tests__/userService.test.ts

```diff
@@ -90,4 +90,37 @@ describe("mongo userService", (): void => {
 
     expect(emptyRes.length).toEqual(0);
   });
+
+  it("getAllTeachers", async () => {
+    const createRes = await UserModel.insertMany(testUsers);
+
+    const testSchools = [
```

Comment:
can you use the testSchools in `testUtils/school.ts` directly? then you don't need to define it again and delete `createRes`

Reply:
I'm not sure if I can use the existing testUsers, because the testUsers don't have an authId, which I need in order to create users in the model. I think these users are intended as return values for mock user service but in this case I need to create the actual users in the database since I'm aggregating.

Reply:
ohhh ok sounds good then